src/components/Heading/Heading.tsx

```diff
@@ -123,6 +123,7 @@ export const Heading = chakra(
         isUppercase,
         isLowercase,
         noSpace,
+        url,
```

Comment:
I forgot but are we marking all components as "prerelease" for the 2.0 update or should we make it a point that some components did not have an update at all? I imagine even if it wasn't a direct update, all components should be bumped up to 2.0, right?

Reply:
This is a question I have been asking myself as well.  And truthfully, I don't know the answer.  There definitely will be some components that are not changed as part of this update and therefore don't need to have the `Latest Version` bumped.  However, this is going to be a major version update, so shouldn't all the components within the release be marked as `v2.0`?